PWGHF/Core/SelectorCuts.h

```diff
@@ -927,6 +927,55 @@ static const std::vector<std::string> labelsPt = {
 static const std::vector<std::string> labelsCutVar = {"m", "CPA", "d0 Jpsi", "d0 Gamma", "pT Jpsi", "pT Gamma", "chi2PCA"};
 } // namespace hf_cuts_chic_to_jpsi_gamma
 
+namespace hf_cuts_sigmac_to_p_k_pi
+{
+static constexpr int nBinsPt = 10;
+static constexpr int nCutVars = 2;
+// default values for the pT bin edges (can be used to configure histogram axis)
+// offset by 1 from the bin numbers in cuts array
+constexpr double binsPt[nBinsPt + 1] = {
+  0.,
+  1.,
+  2.,
+  3.,
+  4.,
+  5.,
+  6.,
+  8.,
+  12.,
+  24.,
+  36.};
+auto vecBinsPt = std::vector<double>{binsPt, binsPt + nBinsPt + 1};
+
+// default values for the cuts
+constexpr double cuts[nBinsPt][nCutVars] = {{0.03, 0.03},  /* 0  < pT < 1  */
```

Comment:
I know that the cut values defined for all the other channels are `double`, however we don't need double precision for them. It is ok for the moment to use `double` for consistency, but I would keep it in mind and modify all of them to `float` in the future.